lib/js/src/rpc/enums/FunctionID.js

```diff
@@ -48,15 +48,50 @@ class FunctionID extends Enum {
     /**
     * @return {Number}
     */
-    static get ADD_COMMAND(){
-        return FunctionID.MAP.AddCommand;
+    static get REGISTER_APP_INTERFACE() {
+        return FunctionID._MAP.RegisterAppInterface;
     }
 
     /**
     * @return {Number}
     */
-    static get ON_LANGUAGE_CHANGE(){
-        return FunctionID.MAP.OnLanguageChange;
+    static get ADD_COMMAND() {
```

Comment:
While I personally believe that enums should always be all-caps, unfortunately doing so would create complications with the library RPC generator if we don't follow the RPC Spec's existing names.

I recommend that we accept this contribution as-is for the sake of early testing, and plan for the RPC generator to overwrite these static getter method names with the proper naming convention when it is ready (prior to an official JavaScript Suite release).

Reply:
@nickschwab it would be good if you could provide us the document with proper naming convention you mentioned here.

Reply:
@vladmu The document is the RPC Spec, available at: https://github.com/smartdevicelink/rpc_spec/blob/master/MOBILE_API.xml

See the [FunctionID enum](https://github.com/smartdevicelink/rpc_spec/blob/master/MOBILE_API.xml#L2774) and related function request definitions (e.g. [AddCommand](https://github.com/smartdevicelink/rpc_spec/blob/master/MOBILE_API.xml#L4922)) in the spec.

Reply:
@nickschwab the Spec has `AddCommandID` value in this case, where is the convention to transform this into `ADD_COMMAND` form?
```
        <element name="AddCommandID" value="5" hexvalue="5" since="1.0" />
```

Reply:
@vladmu Please read over my comments again - I'm not advocating for `ADD_COMMAND`, as that would go against the RPC Spec. The value should either be `AddCommand` (the function name) or `AddCommandID` (the FunctionID enum). Given that this particular class is an enum, I believe it should be `AddCommandID` in order to exactly match the enum definition in the RPC spec. The same logic should apply to all enum classes.

Reply:
It has been a struggle to take the poorly written xml file and write classes with coding standards. These instances can be seen in the other libraries for example here is AddCommand in the Java_Suite library https://github.com/smartdevicelink/sdl_java_suite/blob/master/base/src/main/java/com/smartdevicelink/protocol/enums/FunctionID.java#L51